src/contexts/UnifiedBridgeProvider.tsx

```diff
@@ -476,13 +517,23 @@ export function UnifiedBridgeProvider({
     [t]
   );
 
+  const isBridgeTx = useCallback(
+    (txInfo: IsBridgeTxParams) => {
+      assert(core, CommonError.Unknown);
+
+      return core.isBridgeTx(txInfo);
+    },
+    [core]
+  );
+
   return (
     <UnifiedBridgeContext.Provider
       value={{
         estimateTransferGas,
         getErrorMessage,
         state,
-        getAssetAddressOnTargetChain,
+        isBridgeTx,
+        getAssetIdentifierOnTargetChain,
```

Comment:
should we add tests for these?

Reply:
The `getAssetIdentifierOnTargetChain` will be removed as soon as ICTT is finalized, so I wouldn't bother personally.

`isBridgeTx` is now `analyzeTx` - I'll add tests in the upcoming PR (here: https://github.com/ava-labs/core-extension/pull/57)